tock/hours/models.py

```diff
@@ -24,51 +24,6 @@ class Meta:
         unique_together = ['project', 'hours_per_period']
         ordering = ['project__name']
 
-class Targets(models.Model):
```

Comment:
An excellent argument for why models should always be commented. Congrats on figuring out this was the relevant model